### ISSUE TITLE
max_occurs not set correctly when referencing a group in a complexType

### ISSUE THREAD

**Issue by AllToTheX** [mvantellingen/python-zeep#374]
Version: '1.1.0'
WSDL and script: see attachement.

When a complexType has a reference to a group, with the maxOccurs set in the complexType, the max_occurs for the element is taken from the group instead of the complexType. resulting in a wrongly parsed response. (the element can contain multiple children but only the first is returned due to /xsd/elements/indicators.py line 297)

```xml
<xsd:group name="PVType">
    <xsd:choice>
        <xsd:element name="F8" type="xsd:double" block="#all" nillable="true"/>
        <xsd:element name="Arr" type="ArrayValueType" nillable="true"/>
        <xsd:element name="NoValue" type="NoValueType" nillable="false"/>
    </xsd:choice>
</xsd:group>

<xsd:complexType name="PVSetType">
    <xsd:group ref="PVType" minOccurs="0" maxOccurs="unbounded"/>
</xsd:complexType>
```

[zeep_reference_issue.zip](https://github.com/mvantellingen/python-zeep/files/827197/zeep_reference_issue.zip)

**Comment by AllToTheX** [comment]
So in the above definition, the max_occurs for any `PVType` is set to `1` instead of `unbound`

**Comment by mvantellingen** [comment]
Should be fixed now, will be in 1.4 release
